server/src/immich/controllers/asset.controller.ts

```diff
@@ -86,12 +101,30 @@ export class AssetController {
     return this.service.run(authUser, dto);
   }
 
+  @Post('restore')
+  @HttpCode(HttpStatus.NO_CONTENT)
+  restoreAssets(@AuthUser() authUser: AuthUserDto, @Body() dto: BulkIdsDto): Promise<void> {
+    return this.service.restoreAll(authUser, dto);
+  }
+
```

Comment:
We have:
- `POST /asset/restore` (restore by ids)
- `DELETE /asset` (delete or trash by ids with force: true/false)
- `POST /asset/trash` (trash/restore by userId)

It is a bit confusing imo. 
- We have two endpoints for trash/delete/restore by ids (trash or delete) and restore
- We have two endpoints for trash/delete/restore by userID (delete or restore) and trash.

What do you think about this instead:
// by ids
- `DELETE /asset` (delete or trash by ids) "delete assets" or "trash assets" via `force`
- `POST /asset/restore` (restore by ids) "restore assets"

// by userId
- `POST /asset/trash/empty` (delete by user id) "empty trash"
- `POST /asset/trash/restore` (restore by user id) "restore trash"

Maybe we can add a summary for the new endpoints in the description?